pkgs/development/libraries/gstreamer/vaapi/default.nix

```diff
@@ -5,11 +5,11 @@
 
 stdenv.mkDerivation rec {
   pname = "gst-vaapi";
-  version = "1.16.0";
+  version = "1.16.1";
 
   src = fetchurl {
-    url = "${meta.homepage}/src/${pname}/${pname}-${version}.tar.xz";
-    sha256 = "07qpynamiz0lniqajcaijh3n7ixs4lfk9a5mfk50sng0dricwzsf";
+    url = "${meta.homepage}/src/gstreamer-vaapi/gstreamer-vaapi-${version}.tar.xz";
```

Comment:
Was this renamed?

Reply:
This is just a partial revert of df63b16126a330f89e64aaa4cdd327cdeb0f406f.